source/jormungandr/jormungandr/scenarios/ridesharing/tests/instant_system_tests.py

```diff
@@ -0,0 +1,193 @@
+          "rideSharingAd": {
+            "id": "4eb9a4ba-fb10-4cf0-bf4c-31ada02a2c91",
+            "type": "DRIVER",
+            "from": {
+              "name": "1 Boulevard Volney, Rennes",
+              "lat": 48.1247,
+              "lon": -1.666796
+            },
+            "to": {
+              "name": "9012 Rue du 8 Mai 1944, Bruz",
+              "lat": 48.031951,
+              "lon": -1.74641
+            },
+            "user": {
+              "alias": "Alice M.",
+              "gender": "FEMALE",
+              "imageUrl": "https://dummyimage.com/128x128/B2EBF2/000.png&text=AM",
+              "rating": {
+                "rate": 0,
+                "count": 0
+              }
+            },
+            "price": {
+              "amount": 0,
+              "currency": "EUR"
+            },
+            "vehicle": {
+              "availableSeats": 4
+            }
+          }
+        }
+      ]
+    }
+  ],
+  "url": "https://jky8k.app.goo.gl/?efr=1&apn=com.is.android.rennes&ibi=&isi=&utm_campaign=KISIO&link=https%3A%2F%2Fwww.star.fr%2Fsearch%2F%3FfeatureName%3DsearchResults%26networkId%3D33%26from%3D48.109377%252C-1.682103%26to%3D48.020335%252C-1.743929%26multimodal%3Dfalse%26departureDate%3D2017-12-25T08%253A00%253A00%252B01%253A00"
+}
+
+"""
+
+# https://stackoverflow.com/a/9312242/1614576
+import re
+regex = re.compile(r'\\(?![/u"])')
+fixed = regex.sub(r"\\\\", fake_response)
+
+mock_get = mock.MagicMock(return_value=utils_test.MockResponse(json.loads(fixed), 200, '{}'))
+
+
+def instant_system_test():
+    with mock.patch('requests.get', mock_get):
+
+        instant_system = InstantSystem('dummyUrl', 'dummyApiKey')
+        from_coord = '48.109377,-1.682103'
+        to_coord = '48.020335,-1.743929'
+        datetime_represent = True
+        request_datetime = 1514181600
+
+        ridesharing_journeys = instant_system.request_instant_system(from_coord=from_coord,
+                                                                     to_coord=to_coord,
+                                                                     datetime_represent=datetime_represent,
+                                                                     request_datetime=request_datetime)
+
+        assert len(ridesharing_journeys) == 2
+        assert ridesharing_journeys[0].pickup_place.addr == "9 Allée Rochester, Rennes"
+        assert ridesharing_journeys[0].dropoff_place.addr == "2 Avenue Alphonse Legault, Bruz"
+        assert ridesharing_journeys[0].pickup_date_time == 1514182079
```

Comment:
I think you can use utils tools to have a clearer test, maybe not the same one you use in code :)